efsity/src/main/java/org/smartregister/command/PublishFhirResourcesCommand.java

```diff
@@ -171,32 +177,98 @@ void setProperties(Properties properties) {
     }
   }
 
-  void publishResources() throws IOException, ValidationException, GenerationException {
-    ArrayList<String> resourceFiles = getResourceFiles(projectFolder);
+  String getFileName(String name) {
+    if ((name.endsWith("Register")) || (name.endsWith("Profile"))) {
+      String regex = "([a-z])([A-Z]+)";
```

Comment:
assuming here that we are always converting from camel case

Reply:
can you add that to a docstring for this function?